src/controller/jobmonitor/monitor.go

```diff
@@ -267,10 +267,15 @@ func (w *monitorController) ListQueues(ctx context.Context) ([]*jm.Queue, error)
 		if skippedUnusedJobType(queue.JobName) {
 			continue
 		}
+		// normalize latency, it sometimes return -1
+		var latency int64
+		if queue.Latency > 0 {
```

Comment:
We should pause this PR for further discussion.

Personally, I don't think changing the negative number to 0 is the right approach. The latency should always be positive, and a negative value typically indicates a misconfiguration, such as when the system time on the worker node running harbor-core and harbor-jobservice are not synchronized with the same time server.

Adjusting the value to 0 might mask the underlying issue for the administrator. I prefer that we add a debug log to help identify the potential cause of this situation.